cpp/dolfin/fem/assemble_matrix_impl.cpp

```diff
@@ -41,6 +41,9 @@ void fem::impl::assemble_matrix(Mat A, const Form& a,
   const int num_dofs_per_cell1 = dofmap1.element_dof_layout->num_dofs();
 
   // Prepare constants
+  if (a.list_unset_constants().size() > 0)
```

Comment:
`if (!a.list_unset_constants().empty())`

Reply:
I considered it. "Not unset empty" is a triple negative, and harder to make sense of.

Reply:
Not sure I agree about the triple - parses easy enough for me. `empty` is generally preferred because it is defined (efficiently) for more STL containers.